### DISCUSSION TITLE
Patch response body

### DISCUSSION THREAD

**Discussion by kelseybrennan** [yahoo/elide#2123] [Q&A] [answered]
Hi, is it possible to configure Elide to return the updated object in the response body for a PATCH operation? The JSON-API spec says that the response can either be a 200 OK or a 204 No Content (https://jsonapi.org/format/#crud-updating-responses).

**Comment by aklish** [comment, marked as answer]
I originally thought this was not supported, but it looks like support is there.

If you set the following setting in ElideSettings to 200 (when you configure the Elide bean):

https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/ElideSettings.java#L42

which you can set by calling:

https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/ElideSettingsBuilder.java#L161-L164

It should change the behavior of patch requests to return the response body:

https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/jsonapi/parser/state/BaseState.java#L173-L176

**Reply by aklish** [reply]
If this doesn't work for you, let me know.  I will update the documentation with this feature flag.

**Reply by kelseybrennan** [reply]
That works, thanks!
